source/common/http/conn_manager_utility.cc

```diff
@@ -44,8 +44,8 @@ ServerConnectionPtr ConnectionManagerUtility::autoCreateCodec(
     return ServerConnectionPtr{new Http2::ServerConnectionImpl(
         connection, callbacks, scope, http2_settings, max_request_headers_kb)};
   } else {
-    return ServerConnectionPtr{
-        new Http1::ServerConnectionImpl(connection, callbacks, http1_settings)};
+    return ServerConnectionPtr{new Http1::ServerConnectionImpl(
```

Comment:
nit: as long as you are here, please change to std::make_unique

Reply:
done